README.md

```diff
@@ -26,6 +26,13 @@ To use this dbt package, you must have the following:
 - At least one Fivetran Recurly connector syncing data into your destination. 
 - A **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, **Databricks** destination.
 
+### Databricks Dispatch Configuration
+If you are using a Databricks destination with this package you will need to add the below (or a variation of the below) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+```yml
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+```
```

Comment:
I want to call out the databricks dispatch requirement a bit more.

Reply:
Makes sense. Do we need to add this Databricks mention earlier in the Readme too?

<img width="978" alt="Screen Shot 2022-10-13 at 4 14 48 PM" src="https://user-images.githubusercontent.com/108772760/195720876-eb087540-901e-4e8b-9cce-66967fc1810f.png">

We don't have to change anything to `.circleci/config.yml` for Databricks compatibility?

Reply:
Spark covers our databricks compatibility piece which is in the config.yml. You also do have the Databricks destination piece in your README already.

Reply:
Oh, bleh, I do. All good then.